packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/autofocus/AutoFocusFeature.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera.features.autofocus;
+
+import android.hardware.camera2.CameraCharacteristics;
+import android.hardware.camera2.CaptureRequest;
+import io.flutter.plugins.camera.CameraProperties;
+import io.flutter.plugins.camera.features.CameraFeature;
+
+public class AutoFocusFeature extends CameraFeature<FocusMode> {
+  private FocusMode currentSetting = FocusMode.auto;
+
+  // When we switch recording modes we re-create this feature with
+  // the appropriate setting here.
```

Comment:
I have no idea who/what the "we"s in this sentence refer to. It doesn't even feel like they are referring to the same thing; setting a private field would be done internally to the class, but recreating the feature would be done from outside the class. Please use clear subjects in the comment instead.

"we" is almost always problematic in (non-TODO) comments, as this ambiguity is extremely common; I advise against using it in comments.